### ISSUE TITLE
unable to play message in "zn-CN"

### ISSUE THREAD

**Issue by andreddie81** [dlutton/flutter_tts#64]
Android device Samsung tablet.

Default device language is English.

Added a method for speaking, with a Chinese text taken from plugin website.

Nothing is played, no error is displayed.

```
void _speakMessage() {
    print("before Speaking");
    FlutterPluginTts.setLanguage("zh-CN");
    FlutterPluginTts.speak("你好我是声音播放器");
    print("speaking Done");
  }
```

Both debug messages are showed, but the device is not reproducing the message.

Anything missing here?

Does the device language need to be set to chinese in order to play chinese?

**Comment by dlutton** [comment]
@andreddie81 I just tested the text you provided with the language of zh-CN on an android emulator (Pixel 3 XL) with an API level of 28.  The device is in english and I'm hearing the voice speak the text just fine.  I used the example I provided in this plugin to test it.

**Comment by andreddie81** [comment]
Ok, good to know, do you have any idea or any clue why it is not working on a Samsung tablet? what could be the issue?

**Comment by dlutton** [comment]
I'm not sure.  Have you tried the example in this project on your tablet?  Also, is the tts engine being used supported on the samsung tablet?

**Comment by andreddie81** [comment]
I have actually not used the example provided, I have used the library making it speak in English without any problem, just experienced issues when making it speak in chinese.

**Comment by andreddie81** [comment]
Seems like `FlutterPluginTts.isLanguageAvailable("zh-CN")` returns FALSE on that particular tablet.....any clue on how to add a new language to make it available?

**Comment by dlutton** [comment]
You'd have to check your device's TTS Engine it's using.  This plugin uses `com.google.android.tts`.  Samsung will sometimes use a different one.  You should also call `getLanguages` to check the languages your device supports.

**Comment by andreddie81** [comment]
You are totally right, and all is ok now, sorry for the disturbance. For anyone else with the same problem, in this case, it was Samsung's fault...the default **engine selected is Samsung** and has very limited languages...like 10 or 15.

However, there is also a (slightly hidden menu, at least in Samsung) where the option to **select Google's engine** is available and then the list of languages is huge, and there Chinese is available and works without problem with this plugin.

Thanks!!
